src/launch/policy.c

```diff
@@ -886,10 +886,57 @@ static int policy_export_xmit(Policy *policy, CList *list1, CList *list2, sd_bus
                 "a(buu(" POLICY_T_BATCH "))"                                    \
                 "a(ss)"
 
+static int policy_export_console(Policy *policy, sd_bus_message *m, PolicyEntries *entries, uid_t uid_start, uid_t uid_end) {
```

Comment:
Why uid_t everywhere in this file? I'd prefer if the caller does the conversion. We deal with uint32_t everywhere else here.

Reply:
Done.

Reply:
Agreed, fixed.